launch_ros/launch_ros/actions/node.py

```diff
@@ -51,9 +51,13 @@
 from launch_ros.utilities import normalize_remap_rules
 from launch_ros.utilities import prefix_namespace
 
+import nodl
```

Comment:
If I understand the situation correctly (from [here](https://github.com/ubuntu-robotics/nodl/blob/master/nodl_python/LICENSE)), nodl is under a GPLv3 license.  Because of that, I don't think we can take this into the `launch_ros` core.  I'm going to double-check on this internally, so I'll get back with a definitive answer, but I just wanted to make you aware that this may be a non-starter.

Reply:
Close-- It's LGPL, @clalancette. That's standing guidance from our legal department. Let us know if that proves to be an issue; we can have a conversation with them if necessary, so don't let that stand in the way of review.

Reply:
We're re-licensing as Apache 2.0 in https://github.com/ubuntu-robotics/nodl/pull/35